src/tests/class_tests/smartpeak/source/RawDataProcessor_test.cpp

```diff
@@ -1345,6 +1347,84 @@ BOOST_AUTO_TEST_CASE(pickMS1Features)
   BOOST_CHECK_CLOSE(static_cast<double>(hfeature2.getIntensity()), 2.3054807186126709, 1e-6);
 }
 
+/**
+  PickMS2Features Tests
+*/
+BOOST_AUTO_TEST_CASE(constructorPickMS2Features)
+{
+  PickMS2Features* ptrPickFeatures = nullptr;
+  PickMS2Features* nullPointerPickFeatures = nullptr;
+  BOOST_CHECK_EQUAL(ptrPickFeatures, nullPointerPickFeatures);
+}
+
+BOOST_AUTO_TEST_CASE(destructorPickMS2Features)
+{
+  PickMS2Features* ptrPickFeatures = nullptr;
+  ptrPickFeatures = new PickMS2Features();
+  delete ptrPickFeatures;
+}
+
+BOOST_AUTO_TEST_CASE(gettersPickMS2Features)
+{
+  PickMS2Features processor;
+
+  BOOST_CHECK_EQUAL(processor.getID(), -1);
+  BOOST_CHECK_EQUAL(processor.getName(), "PICK_MS2_FEATURES");
+}
+
+BOOST_AUTO_TEST_CASE(pickMS2Features)
+{
+  // Pre-requisites: load the parameters and associated raw data
+  ParameterSet params_1;
```

Comment:
Can you also add in parameters for storing the convex hull and chromatograms by adding the associated options for `FeatureFinderMetabo`? From the documentation these are "report_convex_hulls" and "report_chromatograms". 

As a follow-up, can you then add tests for the convex hull data and additional chromatogram data?

Reply:
done.